gulpfile.js

```diff
@@ -1149,13 +1155,16 @@ function createGalleryList() {
       }
     });
 
+    var helloWorldIndex = demoObjects.indexOf(helloWorld);
```

Comment:
If someone removes Hello World from their local build, this will return `-1`.  We should set its minimum to zero to protect against this case.